File: SDKs/github.com/ghes-3.1/1.1.4/go/pkg/models/operations/issuesdeletelabel.go
package operations

type IssuesDeleteLabelPathParams struct {
	Name  string `pathParam:"style=simple,explode=false,name=name"`
	Owner string `pathParam:"style=simple,explode=false,name=owner"`
	Repo  string `pathParam:"style=simple,explode=false,name=repo"`
}

type IssuesDeleteLabelRequest struct {
	PathParams IssuesDeleteLabelPathParams
}

type IssuesDeleteLabelResponse struct {
	ContentType string
	StatusCode  int64
}

File: SDKs/github.com/ghes-2.19/1.1.4/python/src/sdk/models/operations/issues_delete_label.py
from dataclasses import dataclass, field



@dataclass
class IssuesDeleteLabelPathParams:
    name: str = field(metadata={'path_param': { 'field_name': 'name', 'style': 'simple', 'explode': False }})
    owner: str = field(metadata={'path_param': { 'field_name': 'owner', 'style': 'simple', 'explode': False }})
    repo: str = field(metadata={'path_param': { 'field_name': 'repo', 'style': 'simple', 'explode': False }})
    

@dataclass
class IssuesDeleteLabelRequest:
    path_params: IssuesDeleteLabelPathParams = field()
    

@dataclass
class IssuesDeleteLabelResponse:
    content_type: str = field()
    status_code: int = field()
    
